src/parser/parser.yy

```diff
@@ -587,12 +587,14 @@ create_tag_sentence
         $$ = new CreateTagSentence($3, new ColumnSpecificationList(), $6);
     }
     | KW_CREATE KW_TAG name_label L_PAREN column_spec_list R_PAREN create_schema_prop_list {
+        CHECK_COLUMN_SPACES(false, $5);
```

Comment:
How about to add another structure, e.g. `ColumnNameList`, for `DROP`?

Reply:
Good idea ! It's better to add a new structure for `DROP`.